### DISCUSSION TITLE
Offer autocompletion for string formatting parameters

### DISCUSSION THREAD

**Discussion by gruvw** [microsoft/pylance-release#4959] [Enhancement]
Issue Type: <b>Bug</b>

# Behaviour

![](https://i.imgur.com/aqjn7AC.png)

## Expected

Not showing the parameters autocompletion.

Extension version: 2022.10.1
VS Code version: Code 1.69.2 (3b889b090b5ad5793f524b5d1d39fda662b96a2a, 2022-07-18T16:14:10.636Z)
OS version: Linux x64 5.15.0-41-generic
Restricted Mode: No

**Comment by judej** [comment]
@guruw, thanks for the report. This is a new feature that we don't have in Pylance today. I will move this to discussions so that others can have an opportunity to provide their opinions and upvote.
